CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Red Hat certified version of the secrets-provider-for-k8s image ([cyberark/secrets-provider-for-k8s#93](https://github.com/cyberark/secrets-provider-for-k8s/pull/93))
 
 ### Changed
+- The retry backoff for the Secrets Provider is now constant instead of exponential ([cyberark/secrets-provider-for-k8s#182](https://github.com/cyberark/secrets-provider-for-k8s/pull/182))
```

Comment:
do we have an issue for this? it is best to link to issues in the CHANGELOG rather than PRs.